UnitTests.Deviationlark/PhoneBookTests/UnitTest1.cs

```diff
@@ -0,0 +1,50 @@
+using PhoneBook;
+namespace PhoneBookTests;
+
+[TestClass]
+public class PhoneBookTest
+{
+    [TestMethod]
+    [DataRow("+38978333333")]
+    public void ValidateNumber_IsValidNumber_ReturnTrue(string phoneNumber)
+    {
+        var result = PhoneBookValidation.ValidateNumber(phoneNumber);
+        Assert.IsTrue(result);
+    }
+    [TestMethod]
+    [DataRow("389783333333")]
+    [DataRow("")]
+    public void ValidateNumber_IsInvalidNumber_ReturnFalse(string phoneNumber)
+    {
+        var result = PhoneBookValidation.ValidateNumber(phoneNumber);
+        Assert.IsFalse(result);
+    }
+    [TestMethod]
+    [DataRow("email@gmail.com")]
+    public void ValidateEmail_IsValidEmail_ReturnTrue(string email)
+    {
+        var result = PhoneBookValidation.ValidateEmail(email);
+        Assert.IsTrue(result);
+    }
+    [TestMethod]
+    [DataRow("email..com")]
+    public void ValidateEmail_IsInvalidEmail_ReturnFalse(string email)
```

Comment:
Please correct the name and return value. You are cheking for e mail if it is valid or not. In this case you are testing invalid emails. So the name should be isvalidemail_returnfalse or isInvalidEmail_returnTrue
Name and return value are inconsistent.